dockerfiles/ubuntu/is/Dockerfile

```diff
@@ -65,11 +65,12 @@ ARG USER_GROUP_ID=802
 ARG USER_HOME=/home/${USER}
 # build arguments for WSO2 product installation
 ARG WSO2_SERVER_NAME=wso2is
-ARG WSO2_SERVER_VERSION=5.11.0
+ARG WSO2_SERVER_VERSION=6.0.0
 ARG WSO2_SERVER_REPOSITORY=product-is
 ARG WSO2_SERVER=${WSO2_SERVER_NAME}-${WSO2_SERVER_VERSION}
 ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}
-ARG WSO2_SERVER_DIST_URL=https://github.com/wso2/${WSO2_SERVER_REPOSITORY}/releases/download/v${WSO2_SERVER_VERSION}/${WSO2_SERVER}.zip
+#ARG WSO2_SERVER_DIST_URL=https://github.com/wso2/${WSO2_SERVER_REPOSITORY}/releases/download/v${WSO2_SERVER_VERSION}/${WSO2_SERVER}.zip
+ARG WSO2_SERVER_DIST_URL=https://github.com/wso2/product-is/releases/download/v6.0.0-rc2/wso2is-6.0.0-rc2.zip
```

Comment:
why this is hardcoded. can't we used it from the rc2 tag?

Reply:
We can change like this
`ARG WSO2_SERVER_DIST_URL=https://github.com/wso2/${WSO2_SERVER_REPOSITORY}/releases/download/v${WSO2_SERVER_VERSION}-rc2/${WSO2_SERVER}-rc2.zip`
When I check the code locally, after the pack extracts, its name is wso2is-6.0.0. Since rc2 part is not there it gives an error.

Reply:
@VivekVinushanth shall we point to a pack in the localmachine and get the docker image? So that users can point to their pack in their local machine

Reply:
Addressed with https://github.com/wso2/docker-is/pull/348/commits/675d438b74c80101df0873c45f38a74beeb853b1